docs/cloud-adoption-guide/intermediate-stage/jumpbox-explainer.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Explainer: what is a jumpbox or bastion host?"
+description: Defines the terms 'jumpbox' and 'bastion host'
+author: telmosampaio
+---
+
+# Explainer: what is a jumpbox?
+
+In the foundational adoption stage, you learned about using a single infrastructure-as-a-service (IaaS) virtual machine (VM). Most IaaS workloads will be composed by more than a single VM. They can encompass several VMs running different services, such as a web UI, distributed micro services, and database servers. 
+
```

Comment:
I think you should add a paragraph that defines the problem space that a jumpbox solves here. The problem is that we need to SSH or RDP to secure VMs, but we don't want to put public IP addresses on them. So, we need to have a public IP address that is not connected to a VM on the protected private network.